app/services/document_type_schema.rb

```diff
@@ -1,14 +1,16 @@
 # frozen_string_literal: true
 
 class DocumentTypeSchema
-  attr_reader :fields, :document_type, :name, :supertype, :managed_elsewhere
+  attr_reader :fields, :document_type, :name, :supertype, :managed_elsewhere, :schema_name, :rendering_app
```

Comment:
Maybe we can make these required in the spec: https://github.com/alphagov/content-publisher/blob/master/spec/formats/configuration_spec.rb#L20

Reply:
Except they're actually optional e.g. consultation.

Reply:
Ah yes, that makes sense.